services/prefect/flows/nyt_updater.py

```diff
@@ -0,0 +1,34 @@
+from prefect import Flow
+from prefect.schedules import CronSchedule
+
+from prefect.tasks.prefect import create_flow_run, wait_for_flow_run
+from prefect.utilities.notifications import slack_notifier
+from prefect.engine.state import Failed
+
+
+def init_updater_flow():
+    with Flow(
+        "NYTParquetUpdater",
+        schedule=CronSchedule("30 6 * * *"),
+        state_handlers=[slack_notifier(only_states=[Failed])]  # notify on failure only,
+    ) as flow:
+
+        # NYTimesCasesDeaths and UpdateParquetFiles flows must already be registered via
+        # generated_flows.py and update_api_view.py
+        nyt_flow = create_flow_run(
+            flow_name="NYTimesCasesDeaths", project_name="can-scrape"
+        )
+        wait_for_nyt = wait_for_flow_run(nyt_flow, raise_final_state=True)
+
+        parquet_flow = create_flow_run(
+            flow_name="UpdateParquetFiles", project_name="can-scrape"
+        )
+        wait_for_parquet = wait_for_flow_run(parquet_flow, raise_final_state=True)
+
+        parquet_flow.set_upstream(wait_for_nyt)
```

Comment:
Not 100% sure what this `set_upstreame()` is for, but wondering if it should be `wait_for_parquet` instead of `wait_for_nyt`?

Reply:
I believe what `set_upstream()` does here is make the `parquet_flow` flow wait until `wait_for_nyt` has been completed before executing. `wait_for_nyt` is a task that waits to complete until the `nyt_flow` has finished. I think basically what's it's saying is "wait for the NYT flow to finish before running the Parquet flow".

So, I think that here the `wait_for_parquet` variable actually isn't needed, but it should still be `parquet_flow.set_upstream(wait_for_nyt)`. 

The docs ([here](https://docs.prefect.io/api/latest/core/task.html#task-2) and [here](https://docs.prefect.io/api/latest/tasks/prefect.html#wait-for-flow-run)) give a little bit more context, but they don't give a huge amount of explanation

Reply:
Oh!  I see, yeah, that makes sense.  I got confused because of the ordering (calling `wait_for_flow_run(parquet_flow, ...)` before specifying that the parquet_flow depends on the nyt_flow)...